src/neuroglancer/util/google_oauth2.ts

```diff
@@ -250,9 +250,11 @@ export function authenticateGoogleOAuth2(
   } else {
     if (!cancellationToken.isCanceled) {
       const newWindow = open(url);
-      request.finished.add(() => {
-        newWindow.close();
-      });
+      if (newWindow !== null) {
+        request.finished.add(() => {
+          newWindow.close();
+        });
+      }
```

Comment:
```
    ERROR in ./src/neuroglancer/util/google_oauth2.ts
    [tsl] ERROR in /Users/John/2017/fall/neuroglancer/src/neuroglancer/util/google_oauth2.ts(254,9)
          TS2531: Object is possibly 'null'.
```
Well, it can't be `null` now. It may be worth looking into why this would be `null` and what the sensible response should be.